status/status.go

```diff
@@ -59,6 +59,8 @@ var transitioningMap = map[string]string{
 	"Updating":                    "updating",
 	"Waiting":                     "waiting",
 	"InitialRolesPopulated":       "activating",
+	"ServiceAccountCreated":       "creating",
+	"PrivilegesCreated":           "creating",
```

Comment:
I think `Privilege` is not a proper word, we should change it to `Role` or `ServiceAccountRole`